Gemfile

```diff
@@ -19,61 +19,66 @@ gem "sprockets-rails", "3.4.2"
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 # Use the Puma web server [https://github.com/puma/puma]
-gem "puma", "5.6.5"
+gem 'puma', '5.6.5'
 
 # Application settings
-gem 'config', "4.1.0", github: 'rubyconfig/config'
+gem 'config', '4.1.0', github: 'rubyconfig/config'
 
 # PostgresSQL
 gem 'pg', '1.4.5'
 
 # Use Redis for Caching and Sidekiq
-gem "redis", "5.0.5"
+gem 'redis', '5.0.5'
 
 # Async and Delayed Jobs
-gem "sidekiq", "7.0.2"
+gem 'sidekiq', '7.0.2'
 
 # Full-text ElasticSearch + Chewy
-gem 'chewy', "7.2.7"
+gem 'chewy', '7.2.7'
 
 # User's input sanitization
-gem "sanitize", "6.0.0"
+gem 'sanitize', '6.0.0'
 
 # Pagination
-gem "kaminari", "1.2.2"
+gem 'kaminari', '1.2.2'
 
 # Cron based periodical jobs
-gem "whenever", "1.0.0", require: false
+gem 'whenever', '1.0.0', require: false
 
 # Authentication solution
-gem "devise", "4.8.1"
+gem 'devise', '4.8.1'
 
 group :development, :test do
   # RSpec testing
-  gem "rspec-rails", "6.0.1"
+  gem 'rspec-rails', '6.0.1'
 
   # Fake data for development and testing
-  gem "faker", "3.1.0"
+  gem 'faker', '3.1.0'
 
   # Code linting
-  gem "rubocop-rails", "2.17.4", require: false
+  gem 'guard'
+  gem 'guard-rspec'
+  gem 'letter_opener'
```

Comment:
Letter opener does not work with docker. Not sure how you run and why. 
Project has mailcatcher.